docs/references/configs/page.md

```diff
@@ -78,6 +78,26 @@ This document lists all the configuration options supported by the Gofr framewor
 - Name: CMD_LOGS_FILE
 - Description: File to save the logs in case of a CMD application
 
+---
+
+- Name: STATIC_DIRECTORY_LISTING
+- Description: To enable directory listing of files in folder (default: true)
+
+---
+
+- Name: STATIC_HIDEDOTFILES
```

Comment:
There is no need to serve a hidden file but when I was testing it it was serving by default when using fileServer so I had set it to true to not serve hidden files by default but if someone wants to serve it they can change the configuration to false to render it.

Reply:
It is just an added option, I know that it is absurd to even provide such option but I added it just in case.

Reply:
I don't think we should even give an option, just so we can avoid any accidental exposure of hidden files.

Reply:
@vipul-rawat I agree with you, but I would like to spot something that is somehow standard now

https://en.wikipedia.org/wiki/Well-known_URI
https://github.com/moul/awesome-well-known
https://www.iana.org/assignments/well-known-uris/well-known-uris.xhtml

I'm unsure how people would be able to share such document

Here is a real one
https://accounts.google.com/.well-known/openid-configuration

But yes, it would be a breach to allow sharing "hidden" files

Reply:
by default, I have set the dot files rendering to false

Reply:
@ccoVeille I think this is a matter of `can` and not `must`, this is what I can infer from the documents and I strongly believe that we should avoid cases where users can "accidentally" expose files that were not supposed to be.

Reply:
I do agree, no worry. It's just that maybe the well-known folder could/should be planned as a dedicated feature

Reply:
Yes, we can have a discussion thread for the same -  if we want to support the well-known for all the standard implementations!

Reply:
I opened #856